compiler/rustc_parse/src/parser/item.rs

```diff
@@ -1176,13 +1178,14 @@ impl<'a> Parser<'a> {
     }
 
     /// Parses an enum declaration.
-    fn parse_item_enum(&mut self) -> PResult<'a, ItemInfo> {
+    fn parse_item_enum(&mut self, start_span: Span) -> PResult<'a, ItemInfo> {
```

Comment:
`start_span` seems a little misnomer here, given that the span begins at the `enum` or `struct` keyword, rather than covering the entire item (i.e. including the qualifiers such as `pub`). Maybe something like `keyword_span` would work better?